src/components/PostList.tsx

```diff
@@ -1,38 +1,10 @@
 import React from "react";
-import { useAppSelector } from "../redux/hooks";
-import { FeedItem } from "../utilities/types";
-import { NoteActivityPub } from "../utilities/activityPubTypes";
+import Post from "./Post";
 
 const PostList: React.FC = () => {
-  const feed: FeedItem[] = useAppSelector((state) => state.feed.feed).filter(
-    (post) => post?.content?.type === "Note"
-  );
-
-  const postItems = feed.map((post: FeedItem, index: number) => {
-    const noteContent: NoteActivityPub = post.content as NoteActivityPub;
-    return (
-      <div key={index}>
-        {noteContent && (
-          <>
-            <div>{post.fromAddress}</div>
-            <div>{noteContent.content}</div>
-            {noteContent.attachment && (
-              <img
-                width={400}
-                src={noteContent.attachment[0]?.url}
-                alt={noteContent.attachment[0]?.url ? "Image for post" : ""}
-              />
-            )}
-          </>
-        )}
-      </div>
-    );
-  });
-
   return (
     <div className="PostList__block">
```

Comment:
Even though this is so small and a currently just a wrapper component, there will be more functionality here in the future, so I want to keep the component structure like this.